oval_graph/arf_xml_parser/arf_xml_parser.py

```diff
@@ -0,0 +1,117 @@
+"""
+    This file contains a class for creating OVAL graph from ARF XML source
+"""
+
+import os
+import sys
+
+from lxml import etree as ET
+
+from ..exceptions import NotTestedRule
+from ..oval_tree.builder import Builder
+from ._oval_scan_definitions import _OVALScanDefinitions
+from .global_namespaces import namespaces
+
+
+class ARFXMLParser:
+    def __init__(self, src):
+        self.src = src
+        self.tree = ET.parse(self.src)
+        self.root = self.tree.getroot()
+        self.arf_schemas_src = '../schemas/arf/1.1/asset-reporting-format_1.1.0.xsd'
+        if not self.validate(self.arf_schemas_src):
+            start_red_color = '\033[91m'
+            end_red_color = '\033[0m'
+            message = "{}Warning: This file is not valid arf report.{}".format(
+                start_red_color, end_red_color)
+            print(message, file=sys.stderr)
+        try:
+            self.used_rules, self.not_tested_rules = self._get_rules_in_profile()
+            self.report_data_href = list(self.used_rules.values())[0]['href']
+            self.report_data = self._get_report_data(self.report_data_href)
+            self.definitions = self._get_definitions()
+            self.oval_definitions = self._get_oval_definitions()
+            self.scan_definitions = _OVALScanDefinitions(
+                self.definitions, self.oval_definitions, self.report_data).get_scan()
+        except BaseException as error:
+            raise ValueError(
+                'This file "{}" is not arf report file or there are no results'.format(
+                    self.src)) from error
+
+    @staticmethod
+    def get_src(src):
```

Comment:
You could use [pathlib](https://docs.python.org/3/library/pathlib.html) for dealing with paths, that way you can just declare the local data directory as a `pathlib.Path`, and then do things like `xsd_path = LOCAL_DATADIR / xsd_basename` etc.
`get_src` is not a very good name, and pathlib does what is being done there.